jazzy/distribution.yaml

```diff
@@ -5832,6 +5832,18 @@ repositories:
       url: https://github.com/hatchbed/opensw_ros.git
       version: ros2
     status: developed
+  orbbec_camera:
+    source:
+      type: git
+      url: https://github.com/orbbec/OrbbecSDK_ROS2.git
+      version: v2-main
```

Comment:
Should we have a different name for v2 vs the main branch? Like something similar to behaviortree? https://github.com/ros/rosdistro/blob/master/jazzy/distribution.yaml#L906 both seem to be supported